tests/lib/snaps/store/test-snapd-libvirt-consumer/vm/ping-unikernel.xml

```diff
@@ -11,7 +11,7 @@
   </serial>
   <qemu:commandline>
     <qemu:arg value='-device'/>
-    <qemu:arg value='virtio-net,netdev=n0'/>
+    <qemu:arg value='virtio-net-pci,netdev=n0,bus=pci.0,addr=03.0'/>
```

Comment:
Maybe a small comment with the link to the upstream bug about this?

Reply:
Added!